From 0a24bd3b1b0cf43e21995f18acea5aa86099c763 Mon Sep 17 00:00:00 2001
From: meows <b5c6@protonmail.com>
Date: Wed, 11 Mar 2020 07:18:04 -0500
Subject: [PATCH 1/2] _specs: 1082 status=Final

It has been at least 2 weeks since this doc has been at
Last Call.

Signed-off-by: meows <b5c6@protonmail.com>
---
 _specs/ecip-1082.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/_specs/ecip-1082.md b/_specs/ecip-1082.md
index 51422e0c..a8249390 100644
--- a/_specs/ecip-1082.md
+++ b/_specs/ecip-1082.md
@@ -3,10 +3,10 @@ lang: en
 ecip: 1082
 title: Fork identifier (`forkid`) protocol fork list specification
 author: github.com/meowsbits
-status: Last Call
+status: Final
 type: Standards Track
-discussions-to: https://github.com/ethereumclassic/ECIPs/issues/294
 category: Networking
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/294
 created: 2019-12-18
 license: Apache-2.0
 ---

From b1c48b16d58de8ac6b8a159eef66901d9d80c8af Mon Sep 17 00:00:00 2001
From: meows <b5c6@protonmail.com>
Date: Mon, 23 Mar 2020 07:52:43 -0500
Subject: [PATCH 2/2] feat(examples): update Classic network cases to latest
 config

Current at c573499c8594c8ab2b8a4dd9ec8590f257cc7ae3
at github.com/etclabscore/core-geth, which includes
ECIP1088 installment for all Classic networks.

Signed-off-by: meows <b5c6@protonmail.com>
---
 _specs/ecip-1082.md | 33 ++++++++++++++++++++++++---------
 1 file changed, 24 insertions(+), 9 deletions(-)

diff --git a/_specs/ecip-1082.md b/_specs/ecip-1082.md
index a8249390..8b7b4336 100644
--- a/_specs/ecip-1082.md
+++ b/_specs/ecip-1082.md
@@ -43,7 +43,7 @@ The following data points are valid at time of writing (see ECIP's `created-at`
 ##### Ethereum Classic Mainnet (ETC)
 
 - Genesis Hash: `0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3`
-- Forks: `1150000`,`2500000`,`3000000`,`5000000`,`5900000`,`8772000`,`9573000`
+- Forks: `1150000`,`2500000`,`3000000`,`5000000`,`5900000`,`8772000`,`9573000`,`10500839`
 
 | Head Block Number | `FORK_HASH` | `FORK_NEXT` | RLP Encoded (Hex) |
 | --- | --- | --- | --- |
@@ -67,14 +67,17 @@ The following data points are valid at time of writing (see ECIP's `created-at`
 | __head=8772000__ | FORK_HASH=518b59c6 | FORK_NEXT=9573000 | c984518b59c683921288 |
 | head=8772001 | FORK_HASH=518b59c6 | FORK_NEXT=9573000 | c984518b59c683921288 |
 | head=9572999 | FORK_HASH=518b59c6 | FORK_NEXT=9573000 | c984518b59c683921288 |
-| __head=9573000__ | FORK_HASH=7ba22882 | FORK_NEXT=0 | c6847ba2288280 |
-| head=9573001 | FORK_HASH=7ba22882 | FORK_NEXT=0 | c6847ba2288280 |
+| __head=9573000__ | FORK_HASH=7ba22882 | FORK_NEXT=10500839 | c9847ba2288283a03ae7 |
+| head=9573001 | FORK_HASH=7ba22882 | FORK_NEXT=10500839 | c9847ba2288283a03ae7 |
+| head=10500838 | FORK_HASH=7ba22882 | FORK_NEXT=10500839 | c9847ba2288283a03ae7 |
+| __head=10500839__ | FORK_HASH=9007bfcc | FORK_NEXT=0 | c6849007bfcc80 |
+| head=10500840 | FORK_HASH=9007bfcc | FORK_NEXT=0 | c6849007bfcc80 |
 
 
 ##### Kotti
 
 - Genesis Hash: `0x14c2283285a88fe5fce9bf5c573ab03d6616695d717b12a127188bcacfc743c4`
-- Forks: `716617`,`1705549`
+- Forks: `716617`,`1705549`,`2000000`,`2200013`,`5000000`
 
 | Head Block Number | `FORK_HASH` | `FORK_NEXT` | RLP Encoded (Hex) |
 | --- | --- | --- | --- |
@@ -83,21 +86,33 @@ The following data points are valid at time of writing (see ECIP's `created-at`
 | __head=716617__ | FORK_HASH=a3270822 | FORK_NEXT=1705549 | c984a3270822831a064d |
 | head=716618 | FORK_HASH=a3270822 | FORK_NEXT=1705549 | c984a3270822831a064d |
 | head=1705548 | FORK_HASH=a3270822 | FORK_NEXT=1705549 | c984a3270822831a064d |
-| __head=1705549__ | FORK_HASH=8f3698e0 | FORK_NEXT=0 | c6848f3698e080 |
-| head=1705550 | FORK_HASH=8f3698e0 | FORK_NEXT=0 | c6848f3698e080 |
+| __head=1705549__ | FORK_HASH=8f3698e0 | FORK_NEXT=2000000 | c9848f3698e0831e8480 |
+| head=1705550 | FORK_HASH=8f3698e0 | FORK_NEXT=2000000 | c9848f3698e0831e8480 |
+| head=1999999 | FORK_HASH=8f3698e0 | FORK_NEXT=2000000 | c9848f3698e0831e8480 |
+| __head=2000000__ | FORK_HASH=7faf51b5 | FORK_NEXT=2200013 | c9847faf51b5832191cd |
+| head=2000001 | FORK_HASH=7faf51b5 | FORK_NEXT=2200013 | c9847faf51b5832191cd |
+| head=2200012 | FORK_HASH=7faf51b5 | FORK_NEXT=2200013 | c9847faf51b5832191cd |
+| __head=2200013__ | FORK_HASH=595e180d | FORK_NEXT=5000000 | c984595e180d834c4b40 |
+| head=2200014 | FORK_HASH=595e180d | FORK_NEXT=5000000 | c984595e180d834c4b40 |
+| head=4999999 | FORK_HASH=595e180d | FORK_NEXT=5000000 | c984595e180d834c4b40 |
+| __head=5000000__ | FORK_HASH=b5407fd6 | FORK_NEXT=0 | c684b5407fd680 |
+| head=5000001 | FORK_HASH=b5407fd6 | FORK_NEXT=0 | c684b5407fd680 |
 
 
 ##### Mordor
 
 - Genesis Hash: `0xa68ebde7932eccb177d38d55dcc6461a019dd795a681e59b5a3e4f3a7259a3f1`
-- Forks: `301243`
+- Forks: `301243`,`999983`
 
 | Head Block Number | `FORK_HASH` | `FORK_NEXT` | RLP Encoded (Hex) |
 | --- | --- | --- | --- |
 | head=0 | FORK_HASH=175782aa | FORK_NEXT=301243 | c984175782aa830498bb |
 | head=301242 | FORK_HASH=175782aa | FORK_NEXT=301243 | c984175782aa830498bb |
-| __head=301243__ | FORK_HASH=604f6ee1 | FORK_NEXT=0 | c684604f6ee180 |
-| head=301244 | FORK_HASH=604f6ee1 | FORK_NEXT=0 | c684604f6ee180 |
+| __head=301243__ | FORK_HASH=604f6ee1 | FORK_NEXT=999983 | c984604f6ee1830f422f |
+| head=301244 | FORK_HASH=604f6ee1 | FORK_NEXT=999983 | c984604f6ee1830f422f |
+| head=999982 | FORK_HASH=604f6ee1 | FORK_NEXT=999983 | c984604f6ee1830f422f |
+| __head=999983__ | FORK_HASH=f42f5539 | FORK_NEXT=0 | c684f42f553980 |
+| head=999984 | FORK_HASH=f42f5539 | FORK_NEXT=0 | c684f42f553980 |
 
 
 ##### Morden
